topic2KeyPoint5/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>leaMavenK5</groupId>
+  <artifactId>topic2KeyPoint5</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+
+  <name>topic2KeyPoint5</name>
+
+  <modules>
+  	<module>topic2KeyPoint1AndKeyPoint2</module>
+  	<module>topic2KeyPoint4</module>
+  </modules>
+
+  <url>http://maven.apache.org</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```

Comment:
try to use JUnit version 4.x.x in order to use Annotations in test cases. ! 

This is OK, but is not used any more.